packages/react/src/internal/FeatureFlags.js

```diff
@@ -17,4 +17,4 @@
  * const MyComponent = props => (<div {...props}>{aFeatureFlag ? 'foo' : 'bar'}</div>);
  */
 
-/* Currently no feature flag is in use, but keeping this file as a placeholder */
+export const useControlledStateWithEventListener = false;
```

Comment:
Flipping the switch upon React context is in consideration, for the time once we have finished implementing this mechanism to all form components.